defs.bzl

```diff
@@ -154,6 +154,22 @@ def _parse_artifact_str(artifact_str):
     else:
         return parse.parse_maven_coordinate(artifact_str)
 
+JAVA_EXPORT_REPO_NAME = "internal_java_exports_maven"
+
+def java_export_deps():
+    """
+        Adds the maven dependencies that are needed to run java_export rule.
+    """
+    maven_install(
+        name = JAVA_EXPORT_REPO_NAME,
+        artifacts = [
+            "com.google.cloud:google-cloud-storage:1.111.2",
+        ],
+        repositories = [
+            "https://jcenter.bintray.com/",
```

Comment:
The repo list should be parameterized through `java_export_deps()` for users to switch repos.